collagraph/collagraph.py

```diff
@@ -206,12 +218,16 @@ def update_class_component(self, fiber: Fiber):
         self.reconcile_children(fiber, children)
 
     def update_function_component(self, fiber: Fiber):
-        children = [fiber.type(fiber.props)]
+        type_signature = inspect.signature(fiber.type)
```

Comment:
I'm not too sure about this one... Actually: I should just change this into a check whether the fiber.children is a dict. Should be faster and then you'll get an error if your render function doesn't support slots. Which is what you would expect.